provisioning/ubuntu-provision.sh

```diff
@@ -615,63 +615,6 @@ function cleanup() {
   sync
 }
 
-function sanity_check() {
-  echo "== Sanity Check of installed tools, running as user ${username}"
-  su - "${username}" -c "source ${asdf_install_dir}/asdf.sh \
-  && echo 'git-lfs version:' \
-  && git-lfs --version \
-  && echo 'goss version:' \
-  && goss --version \
-  && echo 'hadolint version:' \
-  && hadolint -v \
-  && echo 'java version:' \
-  && java -version \
-  && echo 'jq version:' \
-  && jq --version \
-  && echo 'jx-release-version version:' \
-  && jx-release-version -version \
-  && echo 'kubectl version:' \
-  && kubectl version --client \
-  && echo 'make version:' \
-  && make --version \
-  && echo 'maven version:' \
-  && mvn -v \
-  && echo 'netlify-deploy version:' \
-  && netlify-deploy --help \
-  && echo 'ssh-agent version:' \
-  && command -v ssh-agent \
-  && echo 'packer version:' \
-  && packer -v \
-  && echo 'parallel version:' \
-  && parallel --version \
-  && echo 'python3 version:' \
-  && python3 --version \
-  && echo 'ruby version:' \
-  && ruby -v \
-  && echo 'terraform version:' \
-  && terraform -v \
-  && echo 'unzip version:' \
-  && unzip -v \
-  && echo 'updatecli version:' \
-  && updatecli version \
-  && echo 'vagrant version:' \
-  && vagrant -v \
-  && echo 'yq version:' \
-  && yq --version \
-  && echo 'zip version:' \
-  && zip -v \
-  && echo 'npm version:' \
-  && npm --version \
-  && echo 'playwright version:' \
-  && playwright --version \
-  && echo 'launchable version:' \
-  && launchable --version
-  "
-  echo "== End of sanity check"
-  echo "== Installed packages:"
-  dpkg -l
```

Comment:
Do you mind keeping this instruction : it is not a sanity check per-se, but it helps for auditing software versions for packages (proposal : run it after calling the function `main()` below)